.github/CODEOWNERS

```diff
@@ -145,7 +144,6 @@ receiver/receivercreator/                            @open-telemetry/collector-c
 receiver/redisreceiver/                              @open-telemetry/collector-contrib-approvers @pmcollins @dmitryax
 receiver/sapmreceiver/                               @open-telemetry/collector-contrib-approvers @owais
 receiver/signalfxreceiver/                           @open-telemetry/collector-contrib-approvers @pjanotti @dmitryax
-receiver/simpleprometheusreceiver/                   @open-telemetry/collector-contrib-approvers @asuresh4
```

Comment:
we should keep track of who should we have here.

Reply:
i added an item to #3870 to identify this as a receiver without code owner